package.json

```diff
@@ -134,6 +134,10 @@
     "typescript": "3.5.2"
   },
   "resolutions": {
+    "@apollo/react-common": "0.1.0-beta.8",
+    "@apollo/react-components": "0.1.0-beta.7",
+    "@apollo/react-hoc": "0.1.0-beta.7",
+    "@apollo/react-hooks": "0.1.0-beta.10",
```

Comment:
Those resolutions are currently necessary as the beta packages do not reference each other in a proper way (https://github.com/apollographql/react-apollo/blob/8b6abe068c3027f2b539f4906f5ba5d7c628090a/packages/all/package.json#L42-L46)